src/org/jitsi/meet/test/util/MeetUIUtils.java

```diff
@@ -554,6 +554,25 @@ public static String getLargeVideoID(WebDriver driver)
     }
 
     /**
+     * Checks whether the large video has changed to the desired resource.
+     * @param driver the driver to check
+     * @param resource the expected resource
+     */
+    public static void assertWrongAvatarOnLarge(
```

Comment:
"assertWrong" feels a bit misleading, like this assertion will check if the avatar actually *is wrong*/different than the tested one. But int the check we're looking if it's *right*(asserting equality).